app/assets/javascripts/discourse/helpers/icon-or-image.js.es6

```diff
@@ -1,14 +1,15 @@
 import { htmlHelper } from "discourse-common/lib/helpers";
-import { iconHTML } from "discourse-common/lib/icon-library";
+import { iconHTML, convertIconClass } from "discourse-common/lib/icon-library";
 
 export default htmlHelper(function({ icon, image }) {
   if (!Ember.isEmpty(image)) {
     return `<img src='${image}'>`;
   }
 
-  if (Ember.isEmpty(icon) || icon.indexOf("fa-") !== 0) {
+  if (Ember.isEmpty(icon) || icon.indexOf("fa-") < 0) {
```

Comment:
This small change lets users use both FA 4.7 icons (fa-icon) and FA 5+ icons (far fa-icon).